### ISSUE TITLE
Feature request: History of completed items

### ISSUE THREAD

**Issue by e111077** [PolymerLabs/project-health#278]
Perf Elliott would love you millions more if we could somehow store tasks / PRs when something goes from blue to gone. It would be super lovely for perf / snippets.

I think I can even do with a localstorage of just today's things so I can copy them down in snippets afterwards.

**Comment by samuelli** [comment]
Yep, I like this idea, probably gonna hold off on this for now though.

General thought: grayed out items at the bottom of each section. Stays around for a day and/or till Sunday of each week.
